### ISSUE TITLE
Is MULTI_NODE_READER_ONLY Volume Capability required?

### ISSUE THREAD

**Issue by leakingtapan** [kubernetes-sigs/aws-ebs-csi-driver#123]
According to [CSI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md):
```
// Can be published as readonly at multiple nodes simultaneously.
MULTI_NODE_READER_ONLY = 3;
```
MULTI_NODE_READER_ONLY should be used for volume that can be published as readonly on multiple nodes, and this is not supported by EBS yet but got [set by the driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/pkg/driver/driver.go#L71)

Should this be removed?

/cc @bertinatto

**Comment by bertinatto** [comment]
Thanks for pointing this out.

I mistakenly assumed it was supported. Just created a PR to remove it.

**Comment by leakingtapan** [comment]
Fixed through #127
